external/llvm-project/mlir/include/mlir/Dialect/Tosa/IR/TosaOps.td

```diff
@@ -139,6 +139,7 @@ def Tosa_Conv3DOp : Tosa_InferShapedTypeOp<"conv3d"> {
     Tosa_IntArrayAttr6:$pad,
     Tosa_IntArrayAttr3:$stride,
     Tosa_IntArrayAttr3:$dilation,
+    OptionalAttr<I64Attr>:$group,
```

Comment:
We have lots of `if (group > 1 && isConv2DOp` added to external in TosaToLinalgNamed lowering that has not been upstreamed.
Therefore, I d like to request a MIGraphX e2e test with `--clone-harness` make sure this change is properly tested -- otherwise we ll have a hard time verifying fused modules from MIGraphX.

Reply:
What is involved in making such a test?  (I've never committed anything to MIGraphX.)  Would it be better to similarly test for "group > 1" in the migraphx conversion and leave the grouped ones out of the rock.conv path?